packages/bubble-dev/start-preset/src/pkg.ts

```diff
@@ -24,7 +24,7 @@ export type TReplacers = {
   },
 }
 
-export const Pkg = (replacers?: TReplacers) => (packagePath: string, shouldAddMissingWorkspace: boolean = true) =>
+export const Pkg = (replacers?: TReplacers) => (packagePath: string, shouldAddMissingWorkspace: boolean = true, templateName: (string) = '') =>
```

Comment:
Why `string` is in parenthesis here?

Reply:
old remainders from allowing it to be null 🙈
removed